backend/tracim_backend/models/data.py

```diff
@@ -106,6 +106,18 @@ def to_string(self, lang, translator) -> Optional[str]:
         if self == self.WEEKLY:
             return translator.get_translation("in the last week", lang)
 
+    def get_value(self, lang, translator) -> Optional[str]:
+        if self == self.INDIVIDUAL:
+            return self.value
+        if self == self.NONE:
+            return self.value
+        if self == self.HOURLY:
+            return translator.get_translation("hourly", lang)
+        if self == self.DAILY:
+            return translator.get_translation("daily", lang)
+        if self == self.WEEKLY:
+            return translator.get_translation("weekly", lang)
```

Comment:
Are these keys translated?
I don't see them added to translation file

Reply:
The translation isn't done yet